ONSdigital/response-operations-ui
# Remove markup from message flashing

# What and why?
The markup import was no longer compatible with newer versions of flask session, so it needed to be removed.

# How to test?

- Deploy
- Run acceptance tests
- Navigate to response-ops, find a secure message conversation thread, send a message, then close and re-open the conversation. Ensure that the correct pop-up messages appear in the box after each action and that the "view conversation" link works.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1177)
